### ISSUE TITLE
Bulk missions for specific ship types flood the menu

### ISSUE THREAD

**Issue by VaelophisNyx** [1010todd/Mega-Freight#14]
so the issue here is that if you have a matching ship type for things like `mission "MGF: EB-1A Forge Coke Import [1]"`, since they check either random % OR if you have the ship, when you DO have the ship it will offer an absolute boat load of these missions all at once, often to the same place
adding a high % random to these would make it offer a lot, but not the absolute deluge of them you get right now

![image](https://github.com/user-attachments/assets/1f258334-26b1-44e3-929d-d784e11fb2a0)
^ offered with just two ships (Jupiters) in the fleet

easy fix is just to change 
![image](https://github.com/user-attachments/assets/54a49f29-863a-45e9-828d-2fbe6ce6baba)

to
![image](https://github.com/user-attachments/assets/e3cd5e51-9b1a-4602-bfc1-8346551be76c)

for example

**Comment by 1010todd** [comment]
Nope, ES logic here is a little unintuitive so the correct unpacking for that is `random < 60 + 'math'` **AND** `"cargo space" > 500` **AND** (`has "flagship attributes: bulk bay"` **OR** `has "ship model: CDB-6K Jupiter"` **OR** `...` **OR** `has "outfit (installed): MGF: Bulk Cargo Bay"`).

It's intentional that that specific station offers massive amount of jobs and you either have a bunch of container ships or I messed up the math stuffs.
